capa/rules/__init__.py

```diff
@@ -647,8 +694,29 @@ def second(s: List[Any]) -> Any:
     return s[1]
 
 
+def parse_flavor(scope: Union[str, Dict[str, str]]) -> Flavor:
+    if isinstance(scope, str):
+        if scope in STATIC_SCOPES:
+            return Flavor(scope, "", definition=scope)
+        elif scope in DYNAMIC_SCOPES:
+            return Flavor("", scope, definition=scope)
+        else:
+            raise InvalidRule(f"{scope} is not a valid scope")
```

Comment:
i'd recommend only supporting rules with both static & dynamic defined for right now. let's handle the extra logic for edge cases later on.